ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/flow-tracking-2.md

```diff
@@ -0,0 +1,72 @@
+# flow-tracking-2
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [Monitoring](#monitoring)
+  - [Flow Tracking](#flow-tracking)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+## Monitoring
+
+### Flow Tracking
+
+#### Flow Tracking Sampled
+
+Sample: 666
+
+Hardware offload for IPv4 traffic: enabled
+
+Hardware offload for IPv6 traffic: enabled
```

Comment:
wondering if it is not worth starting to make a table for this?

Reply:
I meant also the Sample one to have a table with info rather than 3 lines with new lines